src/shared/components/mutationMapper/MutationMapper.tsx

```diff
@@ -35,11 +35,13 @@ import MutationMapperDataStore, {
     PROTEIN_IMPACT_TYPE_FILTER_ID,
 } from './MutationMapperDataStore';
 import WindowStore from '../window/WindowStore';
+import { Mutation } from 'cbioportal-utils';
 
 import styles from './mutationMapper.module.scss';
 
 export interface IMutationMapperProps {
     store: MutationMapperStore;
+    colorDriverVsVus?: boolean;
```

Comment:
should we also support this in the library level by adding this property into default mutation mapper inside `react-mutation-mapper`?

Reply:
I have no opinion on it, whatever you think is best

Reply:
Let's keep it this way for now, we can add support in the library level when we need it.